bootstrap/components/sinks/hdfs-sink-topology-component.json

```diff
@@ -6,7 +6,7 @@
   "streamingEngine": "STORM",
   "fieldHintProviderClass": "com.hortonworks.streamline.streams.cluster.bundle.impl.HDFSBundleHintProvider",
   "transformationClass": "com.hortonworks.streamline.streams.layout.storm.HdfsBoltFluxComponent",
-  "mavenDeps": "org.apache.storm:storm-hdfs:STORM_VERSION^org.slf4j:slf4j-log4j12^org.apache.curator:curator-client^org.apache.curator:curator-framework",
```

Comment:
Do we need to exclude from the hive and hbase source and sink as well ?

Reply:
Actually we can't exclude Avro here since hive-exec "contains" Avro as uber jar. If we want to extract Avro as well we should address submit-tool in Storm to handle classifier as well, and add `hive-exec` as `core` classifier and exclude Avro from there.

Reply:
I've excluded Avro from HBase Sink as well. For Hive sink we still need to work on Storm submit-tool.

Reply:
Never mind. storm-submit-tool is hopefully flexible enough to handle classifier. Addressed in 45e3fa46d729d37447887bf63dcafe7ae6b21f93